awscli/examples/greengrass/create-logger-definition-version.rst

```diff
@@ -0,0 +1,18 @@
+**To create a logger definition version**
+
+The following ``create-logger-definition-version`` example creates a logger definition version and associates it with a logger definition. The version defines four logging configurations: 1) system component logs on the file system of the core device, 2) user-defined Lambda function logs on the file system of the core device, 3) system component logs in Amazon CloudWatch Logs, and 4) user-defined Lambda function logs in Amazon CloudWatch Logs. Note: For CloudWatch Logs integration, your group role must grant appropriate permissions. ::
+
+    aws greengrass create-logger-definition-version \
+        --logger-definition-id "a454b62a-5d56-4ca9-bdc4-8254e1662cb0" \
+        --loggers "[{"Id":"1","Component":"GreengrassSystem","Level":"ERROR","Space":10240,"Type":"FileSystem"},{"Id":"2","Component":"Lambda","Level":"INFO","Space":10240,"Type":"FileSystem"},{"Id":"3","Component":"GreengrassSystem","Level":"WARN","Type":"AWSCloudWatch"},{"Id":"4","Component":"Lambda","Level":"INFO","Type":"AWSCloudWatch"}]"
```

Comment:
Should the outter double quotes being singe quotes instead? Without it, I think the inner double quotes would need to be escaped.

Reply:
You're correct, of course.  :-) 

I'll fix and resubmit.

Reply:
Fixed both instances in the create-logger-definition and create-logger-definition-version command.